crates/ide/src/ide/inlay_hints.rs

```diff
@@ -0,0 +1,93 @@
+use crate::{DefDatabase, FileId};
+use itertools::Itertools;
+use syntax::{SyntaxKind, SyntaxToken, TextRange};
+
+#[derive(Debug, Clone)]
+pub enum InlayHintKind {
+    AttrsetAttribute(String),
```

Comment:
```suggestion
    LetBindingEnd(String), // `= let var`
    AttrsetBindingEnd(String), // `= var`
    RecAttrsetBindingEnd(String), // `= rec var`
```

The current code captures both `LetIn` and `[Rec]Attrset` thus the name is inaccurate. Distinguishing between `let` and normal Attrset does seem informative to me. What do you think?